code/modules/mob/living/simple_animal/abnormality/he/red_shoes.dm

```diff
@@ -0,0 +1,413 @@
+		user.InitializeAIController()
+		user.apply_status_effect(/datum/status_effect/panicked_type/murder)
+
+//BreachEffect and combat
+/mob/living/simple_animal/hostile/abnormality/red_shoes/BreachEffect(mob/living/carbon/human/user)
+	if(!(status_flags & GODMODE))
+		return
+	soundloop.stop()
+	for(var/mob/living/carbon/human/H in GLOB.mob_living_list)//stops possessing people, prevents runtimes. Panicked players are ghosted so use mob_living_list
+		UnPossess(H)
+	if(!possessee)
+		name = "Red Shoe"
+		desc = "The Red Shoes’s bloody enameled leather glistens in the light."
+		icon_state = "redshoes_breach"
+		icon_living = "redshoes_breach"
+		damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.5, WHITE_DAMAGE = 1.5, BLACK_DAMAGE = 1, PALE_DAMAGE = 1.5)
+		new /mob/living/simple_animal/hostile/red_shoe(get_turf(src))
+	toggle_ai(AI_ON)
+	datum_reference.qliphoth_change(-2)
+	status_flags &= ~GODMODE
+	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_ABNORMALITY_BREACH, src)
+	FearEffect()
+
+/mob/living/simple_animal/hostile/abnormality/red_shoes/Found(atom/A)//The solo breach generally sticks together
+	if(istype(A, /mob/living/simple_animal/hostile/red_shoe))
+		var/mob/living/simple_animal/hostile/red_shoe/S = A
+		if(S.stat != DEAD && !S.target && !S.client && faction_check_mob(S))//cannibalized from steel ordeals
+			S.Goto(src,S.move_to_delay,1)
+
+/mob/living/simple_animal/hostile/abnormality/red_shoes/AttackingTarget()
+	. = ..()
+	if(!ishuman(target))
+		return
+	var/mob/living/carbon/human/H = target
+	if(H.stat >= SOFT_CRIT || H.health < 0)
+		ChopFeet(H)
+
+/mob/living/simple_animal/hostile/abnormality/red_shoes/proc/ChopFeet(mob/living/carbon/human/H)
+	var/obj/item/bodypart/l_foot = H.get_bodypart(BODY_ZONE_L_LEG)//Feet are defined asBODY_ZONE_PRECISE_L_FOOT. Does the dismember proc not affect them?
+	var/obj/item/bodypart/r_foot = H.get_bodypart(BODY_ZONE_R_LEG)
+	if(HAS_TRAIT(H, TRAIT_NODISMEMBER))
+		return
+	playsound(src, 'sound/abnormalities/redshoes/RedShoes_Kill.ogg', 100, 1)
+	var/severance = (l_foot?.dismember() && r_foot?.dismember())
+	if(!severance)
+		return
+	H.adjustBruteLoss(500)//kill the target
+
+//***Debuff Definition***/
+//Possession
+/datum/status_effect/red_possess
+	id = "redpossess"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = 120 SECONDS
+	alert_type = /atom/movable/screen/alert/status_effect/red_possess
+	var/sanity_damage_applied = FALSE
+
+/atom/movable/screen/alert/status_effect/red_possess
+	name = "Allure"
+	desc = "You've been possessed by red shoes!"
+	icon = 'ModularTegustation/Teguicons/status_sprites.dmi'
+	icon_state = "allure"
+
+/datum/status_effect/red_possess/on_apply()//causes humans to special panic when applied
+	if(!ishuman(owner))
+		qdel(src)
+		return
+	var/mob/living/carbon/human/H = owner
+	var/mob/living/simple_animal/hostile/abnormality/red_shoes/shoes
+	if(H.sanityhealth == H.maxSanity)//if you're at full sanity
+		H.adjustSanityLoss(50)
+	sanity_damage_applied = TRUE
```

Comment:
Damage is insufficient, you could simply make it 80-120 white damage on application, which would also allow it to be resisted but be scarier early. At the moment it's "Oh no, I'm at max sanity and lost a bit, time to keep regenerating."